scripts/provisioning/s3_start

```diff
@@ -58,6 +58,8 @@ def main():
     elif args.service == 's3bgconsumer':
       # Add action and remove pass
       pass
+    elif args.service == 'haproxy':
+      os.system("/usr/sbin/haproxy -Ws -f /etc/cortx/s3/haproxy.cfg -p /run/haproxy.pid &")
```

Comment:
Also -- is this really `/run/haproxy.pid`?  I previously saw pid files stored in `/var/run/<process>.pid`.  Please check.

Reply:
ok checked myself -- it's /run, closing this comment.